generators/app/templates/src/redux/store.ejs

```diff
@@ -20,6 +20,7 @@ import Navigator from '@components/AppNavigator/navigator';
 import AnalyticsMiddleware from './middlewares/analyticsMiddleware';
 <%_ } _%>
 <%_ if(features.login) { _%>
+
```

Comment:
if it also has firebaseanalytics, then there will be an error of an extra new line

Reply:
😮

Reply:
Nice catch